client/js/constants/constants.js

```diff
@@ -4,12 +4,14 @@ var AppConstants = {
   ViewActionTypes: keyMirror({
     ADD_EVENT: null,
     GET_DAY_DATA: null,
-    GET_MONTH_DATA: null
+    GET_MONTH_DATA: null,
+    STAFF_MOVED: null
   }),
   ServerActionTypes: keyMirror({
     EVENT_ADDED: null,
     DAY_DATA_RECIEVED: null,
-    FETCHED_MONTH_DATA: null
+    FETCHED_MONTH_DATA: null,
+    STAFF_MOVED: null
```

Comment:
Thoughts on having the view action type and server action type both be called STAFF_MOVED. If this is a purposeful design design I would be interested in the reasoning. If not would like to change the server action type to something like staff__schedule_updated